guides/common/modules/proc_creating-custom-provisioning-hooks.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Creating_Custom_Provisioning_Hooks_{context}"]
+= Creating Custom Provisioning Hooks
+
+A custom provisioning snippets allows you to execute custom code during host provisioning.
+You can run code before and/or after the provisioning process.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Templates* and click *Create Template*.
+. In the *Name* field, enter a name for your custom provisioning snippet:
```

Comment:
It should be clear, that the custom snippet hook need to exist. As said, there are many.